README.md

```diff
@@ -59,7 +59,7 @@ Open this folder in your terminal app. And run these commands in the terminal. T
 ## Architecture
 
 - **[Express](https://expressjs.com/)** - Application framework in NodeJS.
-- **[Pug](https://pugjs.org)** - Templating engine for displaying the HTML pages.
+- **[EJS](https://ejs.co/)** - Templating engine for displaying the HTML pages.
```

Comment:
Forgot to update this. Changed to EJS as reads like HTML. Less context switching.